src/Command/RebuildCommand.php

```diff
@@ -163,23 +163,19 @@ private function doDump(AssetInterface $asset, OutputInterface $stdout)
             self::$assetTargetPaths[] = $target;
 
             if (!is_dir($dir = dirname($target))) {
-                $stdout->writeln(sprintf(
-                    '<comment>%s</comment> <info>[dir+]</info> %s',
-                    date('H:i:s'),
-                    $dir
-                ));
+                if (OutputInterface::VERBOSITY_VERBOSE <= $stdout->getVerbosity()) {
+                    $stdout->writeln(sprintf(
+                        '<comment>%s</comment> <info>[dir+]</info> %s',
+                        date('H:i:s'),
+                        $dir
+                    ));
+                }
 
                 if (false === @mkdir($dir, 0777, true)) {
                     throw new \RuntimeException('Unable to create directory '.$dir);
                 }
             }
 
-            $stdout->writeln(sprintf(
```

Comment:
Deleting this log is intentional? Instead of wrapping it with the check of the log level?

Reply:
Yeah, it was intentionally moved (not deleted, per say) to below line 222, so that we only print this info if we are indeed recompiling the asset. Note, I intentionally kept it printing even when verbosity is normal. The goal being for the rebuild case, we will only output file information when:
1.) The file is being (re)-compiled, not from cache
or 
2.) Verbosity is turned up above "normal"

So for most rebuild cases, it should only output ERROR information, and new files compiled.

Reply:
Makes sense! Thank you for the explanation!